tests/test_local.py

```diff
@@ -23,6 +23,16 @@
 SDIR = os.path.dirname(os.path.abspath(__file__))
 
 
+class TestLocalPopen:
```

Comment:
Can you stick a Python 3 guard here, please?

Also, this can just be a function, rather than a class in a function (up to you).

Reply:
Sorry I have to ask for Python 3 guards in 2020, but plumbum still supports Python 2.6. :/ But hey, we might be able to drop Python 2.6 after November 2020 (RHEL 6)!

Reply:
It it okay like this? Or do you want me to put a run-time check on the actual methods?

(I don't know what would be appropriate behaviour then, I guess just raising `NotImplemented` instead of `AttributeError`?)

Reply:
This is fine, adding features to Python 2 is not needed, it was an error before too, I believe.